cryptofeed/exchange/bequant.py

```diff
@@ -74,6 +73,290 @@ def __reset(self):
         self.l2_book = {}
         self.seq_no = {}
 
+    async def _ticker(self, msg: dict, ts: float):
```

Comment:
why does it show all this code being added? This code is already present in the repo

Reply:
Hmm. I think I might know why. I rebased on my 'bequant' branch when I should have rebased on origin (which merged the bequant commits). If you're happy with the changes in principle, reject this one and I'll start afresh on the current master